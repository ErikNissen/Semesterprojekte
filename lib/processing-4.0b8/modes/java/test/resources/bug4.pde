version https://git-lfs.github.com/spec/v1
oid sha256:3b5ca38c74c4dc58209679feb747dae03e708cd7850d14dbc9643c3e70faf4db
size 68
