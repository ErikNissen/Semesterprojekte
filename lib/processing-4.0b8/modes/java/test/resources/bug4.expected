version https://git-lfs.github.com/spec/v1
oid sha256:cb15fae88488e88ac291f692a557f466dfb093d2a109147e9463450058bdc60e
size 829
